static/app/components/events/interfaces/spans/dragManager.tsx

```diff
@@ -109,6 +109,8 @@ class DragManager extends React.Component<DragManagerProps, DragManagerState> {
         return;
       }
 
+      PerformanceInteraction.startInteraction('SpanTreeDrag');
```

Comment:
```suggestion
      PerformanceInteraction.startInteraction('SpanTreeHandleDrag');
```
I think this may make it more clear what specific event occurred if somebody saw this on Sentry, without being familiar with the code (although it's not consistent with the method's name)